doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -135,6 +137,130 @@ The string "ISO8601" as type for attributes and metadata is also supported. The
 
 [Top](#top)
 
+## `dateModified` and `dateCreated` attributes
+
+Section "Attribute names restrictions" establishes that system/builtin attribute names cannot be used as user attribute names. Thus,
+`dateCreated` and `dateModified` (which are system/builtin attributes) cannot be used as user attribute names.
+
+However, due to legacy reasons, Orion doesn't reject attemps of using these attribute with 400 Bad Request errors. **You are
+strongly encouraged to not use dateCreate/dateModified as attribute names** but, if you at the end need to do so, take into account
+the following information.
+
+You can use dateModified/dateCreated attributes at creation/update time, eg:
```

Comment:
e.g.

Reply:
Fixed in 1ab8d0a878a633c982dbe2f016168e73233d3f1b